templates/cluster-template-without-lb.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         nodeRegistration:
           kubeletExtraArgs:
             cloud-provider: external
+            provider-id: openstack:///'{{ instance_id }}'
```

Comment:
Any idea why we're not inheriting this from default? The answer to this question would be outside the scope of this PR, though.

Reply:
This is the generated template! So it is actually inherited from default and that is why we see it here. :slightly_smiling_face: The without-lb patch is [over here](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/kustomize/v1alpha7/without-lb/patch-without-lb.yaml) and does not have any provider-id set.